internal/fs/fs.go

```diff
@@ -337,3 +360,13 @@ func IsRegular(name string) (bool, error) {
 	}
 	return true, nil
 }
+
+// IsSymlink determines if the given path is a symbolic link.
```

Comment:
It should behave in the same way. `os.Lstat()` call [`GetFileAttributesEx`](https://msdn.microsoft.com/en-us/library/windows/desktop/aa364946(v=vs.85).aspx) on Windows and `FileStat.Mode()` will convert the attribute `syscall.FILE_ATTRIBUTE_REPARSE_POINT` to `os.ModeSymlink`. (see [os/stat_windows.go](https://github.com/golang/go/blob/master/src/os/stat_windows.go#L177) and [os/types_windows.go](https://github.com/golang/go/blob/master/src/os/types_windows.go#L41)).

Unless I'm missing something else. 😄